packages/core-api/README.md

```diff
@@ -0,0 +1,22 @@
+# @backstage/core-api
+
+This package provides the core API used by Backstage plugins and apps.
+
+## Installation
+
+Do not install this package directly, it is reexported by `@backstage/core`. Install that instead:
+
+```sh
+$ npm install --save @backstage/core
```

Comment:
should we even mention the npm possibility?

Reply:
Meh, people can use npm if they want. You could set up a new repo, or just switch to npm after scaffolding an app